### PR TITLE
Add license file

### PR DESCRIPTION
**Pull request by vinkla**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015, Jason Lewis
```
Comment (GrahamCampbell) [inline LICENSE]: The year should be as low as possible.

Comment (vinkla) [inline LICENSE]: Should we maybe do a 2014-2015 instead?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+modification, are permitted provided that the following conditions are met:
+
+* Redistributions of source code must retain the above copyright notice, this
+  list of conditions and the following disclaimer.
+
+* Redistributions in binary form must reproduce the above copyright notice,
+  this list of conditions and the following disclaimer in the documentation
+  and/or other materials provided with the distribution.
+
+* Neither the name of tewtr1we nor the names of its
```
Comment (jasonlewis) [inline LICENSE]: `tewtr1we`?

File: LICENSE (hunk 1 above)
Comment (jasonlewis) [inline LICENSE]: Either should be fine.

File: LICENSE (hunk 2 above)
Comment (vinkla) [inline LICENSE]: Fixed.

**Pull request merged**